atlasdb-client/src/main/java/com/palantir/atlasdb/stream/AbstractPersistentStreamStore.java

```diff
@@ -49,7 +49,7 @@ protected AbstractPersistentStreamStore(TransactionManager txManager) {
         super(txManager);
     }
 
-    private final void storeMetadataAndIndex(final long streamId, final StreamMetadata metadata){
+    protected final void storeMetadataAndIndex(final long streamId, final StreamMetadata metadata) {
```

Comment:
There are several cases like this where a now protected method is called in generated code. I can add comments for them, but I'm not sure they're needed, since the protected modifier should communicate already that changes in behavior could break others.